meta-facebook/yv35-rf/src/platform/plat_gpio.c

```diff
@@ -100,10 +88,9 @@ GPIO_CFG plat_gpio_cfg[] = {
 	  NULL },
 	/** Group F: 40-47 **/
 	{ CHIP_GPIO, 40, ENABLE, DISABLE, GPIO_INPUT, GPIO_LOW, PUSH_PULL, GPIO_INT_DISABLE, NULL },
```

Comment:
This should be open drain output, not changed here but should fix it in this PR anyway

Reply:
OK, I think it is OK to fix it in this PR.

Reply:
Done.